app/cloud.go

```diff
@@ -232,18 +232,23 @@ func (a *App) SendNoCardPaymentFailedEmail() *model.AppError {
 }
 
 // Create/ Update a subscription history event
-func (a *App) SendSubscriptionHistoryEvent(userID string) (*model.SubscriptionHistory, error) {
+func (a *App) SendSubscriptionHistoryEvent(userID string) {
```

Comment:
Even though the calling function does not want to block anymore, I still think it is worth returning at least the error. Then, instead of invoking this as a goroutine directly, you can do something like

```go
go func() {
    err := a.SendSubscriptionHistoryEvent(ruser.Id)
    if err != nil {
        logErrorHere(err)
    }
}()
```

This also lets us keep the tests as they are, if I understand correctly, since the tests call this method directly.